languages/go/exercises/concept/using-map/.docs/introduction.md

```diff
@@ -0,0 +1,10 @@
+`map` in go is a key-value type. In other programming language, you might know map as `dict` or associative array. The value of an uninitialized map is `nil`, and you can define map as follows
+
+`make(map[string]int)`
+
+or
+
+```go
+  var foo map[string]int
```

Comment:
I don't see the different between the above example and this example, except for the fact that the variable is defined upfront here. I would suggest to remove one of the examples (probably the above) and maybe even shorten this a bit and include a comment that emphasizes which type is the key and which type is the value:

```suggestion
// Define a map with string keys and int values
empty_map = make(map[string]int)
```

Reply:
I'm wondering if we should give any code examples in the introduction...

Reply:
Ok... seems that has changed since the last time I checked:

> Code examples should only be used to introduces new syntax (students should not need to search the web for examples of syntax). In other cases provide descriptions or links instead of code.

Reply:
:)